### PR TITLE
jazz2-resurrection: Add version 0.7.3

### PR DESCRIPTION
**Pull request by RavenMacDaddy**
Jazz² Resurrection is an open-source reimplementation of the game Jazz Jackrabbit 2.

**Author's note regarding the mandatory repetition of the import-process upon every update:**

I successfully persisted the corresponding folders that end up containing all the game data, but upon simulating an update through Scoop, and then running Jazz² Resurrection again, it still ended up showing the message "Installation is corrupted!", hence why the `Notes` and `Persist` sections are as they are.

## What package release type is this?

> - [x] stable
> - [ ] beta
> - [ ] dev/nightly/canary

### CODE REVIEW

File: bucket/jazz2-resurrection.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+    "version": "0.7.3",
+    "description": "Jazz² Resurrection is an open-source remaster of the game Jazz Jackrabbit 2",
```
Comment (Calinou) [inline bucket/jazz2-resurrection.json:3]: It's a reimplementation, as it doesn't use any of the original code (which was never released):

```suggestion
    "description": "Open source reimplementation of the game Jazz Jackrabbit 2",
```

**Comment by RavenMacDaddy** [comment]
I saw something that I wanna try. 

In the video tutorial it says "compressing content into.\Content\main.dz".

I'll test if I can automate the update still.

**Comment by RavenMacDaddy** [comment]
Didn't work, simply because the Main.dz file gets replaced when importing, which makes the persist disappear.

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**